cmndcntr/show-list-app
# add month headings to listing page

this technique can be use for days, months, or years.

testing instructions:

- insert a few shows in different months
- verify that a month heading displays inbetween months
